typescript/infra/config/environments/mainnet3/owners.ts

```diff
@@ -130,18 +131,32 @@ export const icas: Partial<
   // ----------------------------------------------------------
   // lumia: '0x418E10Ac9e0b84022d0636228d05bc74172e0e41',
```

Comment:
does having this commented out result in us trying to transfer ownership to the deployer key?